### PR TITLE
Impl Series.skew()

### PR DESCRIPTION
**Pull request by Rubtsowa**
name | nthreads | type | size | median | min | max | compile | boxing
-- | -- | -- | -- | -- | -- | -- | -- | --
Series.skew | 1 | Python | 100000000 | 5.041041 | 4.553035 | 6.009044 | NaN | NaN
  |   | SDC | 100000000 | 1.082 | 1.066 | 1.091 | 0.205984 | 0.00001
Series.skew | 4 | Python | 100000000 | 6.681195 | 4.275043 | 14.061077 | NaN | NaN
  |   | SDC | 100000000 | 1.07 | 0.99 | 1.135 | 0.329002 | 0.00001

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -5725,3 +5725,89 @@ def sdc_pandas_series_groupby_impl(self, by=None, axis=0, level=None, as_index=T
...
+    def sdc_pandas_series_skew_impl(self, axis=None, skipna=None, level=None, numeric_only=None):
+        if axis != 0 and axis is not None:
+            raise ValueError('Parameter axis must be only 0 or None.')
+
+        if skipna is None:
+            _skipna = True
+        else:
+            _skipna = skipna
+
+        infinite_mask = numpy.isfinite(self._data)
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py:5786]: No. that's kills performance and scalability

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 2]
```diff
@@ -5725,3 +5725,89 @@ def sdc_pandas_series_groupby_impl(self, by=None, axis=0, level=None, as_index=T
...
+        len_val = len(infinite_mask)
+        data = self._data[infinite_mask]
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_functions.py:5788]: That's too, actually

**Comment by AlexanderKalistratov** [comment]
@Rubtsowa looks good. Could you please remeasure performance?

**Comment by Rubtsowa** [comment]
name | nthreads | type | size | median | min | max | compile | boxing
-- | -- | -- | -- | -- | -- | -- | -- | --
Series.skew | 1 | Python | 100000000 | 9.940206 | 5.385197 | 19.73827 | NaN | NaN
  |   | SDC | 100000000 | 0.989 | 0.711 | 1.241 | 0.167002 | 0
Series.skew | 4 | SDC | 100000000 | 0.731 | 0.719 | 0.847 | 0.105322 | 0.000417

**Comment by Rubtsowa** [comment]
name | nthreads | type | size | median | min | max | compile | boxing
-- | -- | -- | -- | -- | -- | -- | -- | --
Series.skew | 1 | Python | 100000000 | 6.973869 | 4.221513 | 9.153432 | NaN | NaN
  |   | SDC | 100000000 | 0.597 | 0.595 | 0.604 | 0.12701 | 0.00008
Series.skew | 4 | SDC | 100000000 | 0.199 | 0.197 | 0.208 | 0.134013 | 0.000026

File: sdc/functions/numpy_like.py [hunk 3]
```diff
@@ -967,3 +967,77 @@ def getitem_by_mask_impl(arr, idx):
...
+            if numpy.isfinite(arr[idx]):
+                nfinite += 1
+                _sum += arr[idx]
+                square_sum += arr[idx] ** 2
+                cube_sum += arr[idx] ** 3
+            else:
+                return numpy.nan
+
+        n = nfinite
+        m2 = (square_sum - _sum * _sum / n) / n
```
Comment (AlexanderKalistratov) [inline sdc/functions/numpy_like.py:1002]: I believe we could move this formula in somewhere and then use in Series/Rolling/GroupBy methods

@densmirn ?

**Comment by Rubtsowa** [comment]
name | nthreads | type | size | median | min | max | compile | boxing
-- | -- | -- | -- | -- | -- | -- | -- | --
Series.skew(skipna=False) | 1 | Python | 100000000 | 3.396047 | 3.068078 | 4.135047 | NaN | NaN
  |   | SDC | 100000000 | 0.165 | 0.165 | 0.171 | 0.200001 | 0.000002
  | 4 | SDC | 100000000 | 0.08 | 0.078 | 0.09 | 0.211993 | 0.000009

File: sdc/functions/numpy_like.py [hunk 4]
```diff
@@ -967,3 +967,79 @@ def getitem_by_mask_impl(arr, idx):
...
+
+@sdc_overload(skew)
+def np_skew(arr):
+    if not isinstance(arr, types.Array):
+        return
+
+    def skew_impl(arr):
+        len_val = len(arr)
+        n = 0
+        _sum = 0
```
Comment (AlexanderKalistratov) [inline sdc/functions/numpy_like.py:988]: Let's initialize `_sum`, `square_sum` and `cube_sum` with float values `0.`

File: sdc/tests/tests_perf/test_perf_series.py [hunk 5]
```diff
@@ -135,6 +135,8 @@ def _test_case(self, pyfunc, name, total_data_length, input_data=None, data_num=
     TC(name='shape', size=[10 ** 7], call_expr='data.shape', usecase_params='data'),
     TC(name='shift', size=[10 ** 8]),
     TC(name='size', size=[10 ** 7], call_expr='data.size', usecase_params='data'),
+    TC(name='skew', size=[10 ** 8], params='skipna=True'),
+    TC(name='skew', size=[10 ** 8], params='skipna=False', input_data=[test_global_input_data_float64[0]]),
```
Comment (AlexanderKalistratov) [inline sdc/tests/tests_perf/test_perf_series.py:139]: I don't think using `test_global_input_data_float64[0]` is actually a good idea. It contains max_float values which makes impossible to compare results and may cause some other issues.

File: sdc/functions/numpy_like.py (hunk 4 above)
Comment (Rubtsowa) [reply sdc/functions/numpy_like.py:988]: this will lead to errors

Comment (AlexanderKalistratov) [reply sdc/functions/numpy_like.py:988]: what kind of errors?

File: sdc/functions/static.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (AlexanderKalistratov) [inline sdc/functions/static.py:1]: Let's name this file 'statistics'

File: sdc/functions/statics.py [hunk 7]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (AlexanderKalistratov) [inline sdc/functions/statics.py:1]: `statistics` not `statics`

**Pull request merged**